runtests.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
I suggest just move the runtests.py file into tests, and call it as "./tests/runtests.py" from the travis file.